locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgstr ""
+"Você pode criar seus próprios tokens digitais na Hathor Network com apenas "
+"alguns cliques. Eles irão possuir as mesmas características técnicas de alta "
+"escalabilidade e consenso descentralizado dos tokens nativos HTR. O seu "
+"token será independente do token nativo HTR e não haverá relação de preço "
+"entre eles."
+
+#: src/screens/CustomTokens.js:52
+msgid ""
+"Every custom token has a unique <strong>Configuration String</strong> which "
+"must be shared with all other people that will use the custom token."
+msgstr ""
+"Todo token possuí uma <strong>Configuração</strong> que é única e deve ser "
+"enviada para todas as pessoas que irão utilizar esse token."
+
+#: src/screens/CustomTokens.js:53
+msgid ""
+"If you want to use a custom token that already exists, you need to register "
+"this token in your Hathor Wallet. For this, you will need the custom token's "
+"Configuration String, which you can get from the creators of the token."
+msgstr ""
+"Se você quiser usar um token que já existe, você deve registrá-lo na sua "
+"Hathor Wallet. Para isso, você irá precisar da Configuração desse token, a "
+"qual será fornecida pelo criador do token."
+
+#: src/screens/CustomTokens.js:55
+msgid "Create a new token"
+msgstr "Criar um novo token"
+
+#: src/screens/CustomTokens.js:56
+msgid "Register a token"
+msgstr "Registrar um token"
+
+#: src/screens/CustomTokens.js:59
+msgid "Token registered with success!"
+msgstr "Token registrado com sucesso!"
+
+#: src/screens/DashboardTx.js:127
+msgid "Blocks |fn:(See all blocks)|"
+msgstr "Blocos |fn:(Ver todos)|"
+
+#: src/screens/DashboardTx.js:134
+msgid "Transactions |fn:(See all transactions)|"
+msgstr "Transações |fn:(Ver todas)|"
+
+#: src/components/Navigation.js:47 src/screens/DashboardTx.js:154
+msgid "Explorer"
+msgstr "Explorer"
+
+#: src/screens/DashboardTx.js:155
+msgid ""
+"Here you can see the most recent transactions and blocks of the network."
+msgstr ""
+"Aqui você pode visualizar os blocos e transações mais recentes da rede."
+
+#: src/components/TokenHistory.js:242 src/screens/DashboardTx.js:161
+msgid "ID"
+msgstr "ID"
+
+#: src/screens/DashboardTx.js:162
+msgid "Timestamp"
+msgstr "Data e Hora"
+
+#: src/screens/DecodeTx.js:91
+msgid "Decode tx"
+msgstr "Decodificar transação"
+
+#: src/screens/DecodeTx.js:91
+msgid "push"
+msgstr "forçar"
+
+#: src/screens/DecodeTx.js:91
+msgid ""
+"Write your transaction in hex value and click the button to get a human "
+"value description"
+msgstr ""
+"Escreva sua transação em hexadecimal e clique no botão para obter os detalhes"
+
+#: src/screens/DecodeTx.js:93
+msgid "Could not decode this data to a transaction"
+msgstr "Não foi possível decodificar os dados dessa transação"
+
+#: src/screens/LoadWallet.js:115
+msgid "Write the 24 words of your wallet (separated by space)."
+msgstr "Escreva as 24 palavras da sua wallet (separadas por espaço)."
+
+#: src/screens/LoadWallet.js:116
+msgid "Words separated by single space"
```

Comment:
I think we're already accepting multiple spaces and space-like characters. Maybe we can rephrase as "Words separated by spaces" (could also use "spaces" on the line above, singular sounds  a bit off, not sure if wrong though).